nextflow.config

```diff
@@ -29,3 +27,13 @@ profiles{
         }
     }
 }
+
+plugins {
+    id 'nf-prov'
+  }
+  
+prov {
+    enabled = true
+    overwrite = true
+    file = "${params.outDir}/manifest.json"
```

Comment:
This seems like we need an "outDir" parameter to your workflow. Where would manifest.json be written to if we left out the parameter?

Reply:
`params.outDir` is set in [main.nf line 8](https://github.com/mc2-center/nf-histoqc/blob/513b6d6f052f80fcbc60cf2fab97ea8335dfcd93/main.nf#L8C1-L8C28